ui/pages/DataManagement/reducers.js

```diff
@@ -53,29 +54,49 @@ export const uploadQcPipelineOutput = values => submitRequest(
 
 export const deleteEsIndex = index => submitRequest('delete_index', RECEIVE_ELASTICSEARCH_STATUS, { index })
 
-export const uploadRnaSeq = values => (dispatch) => {
+const loadMultipleData = (path, getUpdateData, dispatchType, formatSuccessMessage) => values => (dispatch) => {
   let successResponseJson = null
   return new HttpRequestHelper(
-    '/api/data_management/update_rna_seq',
+    `/api/data_management/${path}`,
```

Comment:
Moving the full URL into the path parameter can make the function easier to be used for more cases.